src/foa-decoder.js

```diff
@@ -26,7 +26,7 @@ var FOARotator = require('./foa-rotator.js');
 var FOAPhaseMatchedFilter = require('./foa-phase-matched-filter.js');
 var FOAVirtualSpeaker = require('./foa-virtual-speaker.js');
 var FOASpeakerData = require('./foa-speaker-data.js');
-
+var Utils = require('./utils');
```

Comment:
Can we add '.js' at the end of file path? I know it's not required, but I want to keep them consistent.